### DISCUSSION TITLE
Docs

### DISCUSSION THREAD

**Discussion by unamed12** [icon-project/btp#16] [General]
- [Blockchain Transmission Protocol 1.1](https://docs.google.com/document/d/1fL_fYeIrH7YoRdJZseFZJy1lXLDyiZezb-isMisONsI/edit?usp=sharing) 
- [BTP Ecosystem Working Group](https://docs.google.com/document/d/1fL_fYeIrH7YoRdJZseFZJy1lXLDyiZezb-isMisONsI/edit?usp=sharing)
- [BTP IIP](https://github.com/icon-project/IIPs/blob/master/IIPS/iip-25.md)
- [BTP 1.0 Design](https://drive.google.com/file/d/12FsFRNnd7IUf6VHdhcJi81W782f36pJA/view?usp=sharing)
- [BTP Relay System](https://drive.google.com/file/d/1UMIpxLbY8Feyx7E7hsizfNtMq23nc6hr/view?usp=sharing)
- [Fee Aggregation SCORE Spec](https://drive.google.com/file/d/1Z5agpWHcB1Va7I6Cg9YoGMmfGzUuhgwi/view?usp=sharing)

**Comment by cow-hs** [comment]
you should change wrong link Blockchain Transmission Protocol 1.1 to https://docs.google.com/document/d/1CgXx1y7vqidJDAnK4g0KQb-5qHSnH6GS/edit#heading=h.gjdgxs
